vaadin-time-picker-flow-testbench/src/main/java/com/vaadin/flow/component/timepicker/testbench/TimePickerElement.java

```diff
@@ -176,7 +176,7 @@ public void waitUntilDropDownOpen() {
      * Closes the drop down for the time picker.
      */
     public void closeDropDown() {
-        executeScript("arguments[0].close()",
+        executeScript("const cb = arguments[0]; window.requestAnimationFrame(function(){ cb.close(); });",
```

Comment:
The `close()` method was being called slightly too early so that it didn't always close. I think it was only just being opened (not yet open) when `close()` was being called, sometimes resulting in the drop down staying open.